src/strings.cpp

```diff
@@ -628,6 +628,12 @@ static int DeterminePluralForm(int64 count, int plural_form)
 		 *  Scottish Gaelic */
 		case 13:
 			return ((n == 1 || n == 11) ? 0 : (n == 2 || n == 12) ? 1 : ((n > 2 && n < 11) || (n > 12 && n < 20)) ? 2 : 3);
+
+		/* Three forms: special case for numbers ending in 00 or [2-9][0-9].
+		 * Used in:
+		 *   Romanian */
+		case 14:
+			return n == 1 ? 0 : (n == 0 || (n % 100 > 0 && n % 100 < 20)) ? 1 : 2;
```

Comment:
Doesn't seem to match the comment.
3 cases, as I read the code, are
- 1
- 0, *1-*19
- everything else

Reply:
My comment doesn't include the first plural, but everything checks out - the code is used in gettext and it's correct.

First plural form:
```
7 beers = 7 beri
314 beers = 314 beri
```

Reply:
Wait, there are three noun forms, not three plural forms. First, there's the singular form, then two plural forms.

Reply:
The code matches http://docs.translatehouse.org/projects/localization-guide/en/latest/l10n/pluralforms.html?id=l10n/pluralforms#l
But the comment is off. Only "0" is special, not numbers ending in "00".

Reply:
I see what you meant now. The ending in `[2-9][0-9]` refers to the final two digits as a regular expression. These have the same plural form as all numbers ending in 00 (100, 1000, 100000, etc).

And as a Romanian native I confirm it all works out as exemplified here: https://output.jsbin.com/kagomig